static/js/main.js

```diff
@@ -55,15 +59,14 @@ function refresh() {
     After, a check needs to be done with existing table elements to see if they were all updated.
     If they weren't, gray out the table box.  If they were, update the data, if new nodes exist add them.
     */
-    newxBeeArray = getNodes();
     var alength = newxBeeArray.length;
     var length = xBeeArray.length;
     var found = false;
     var xBeeOBJ = '';
     for (var i = 0; i < length; i++) {
         xBeeArray[i].ConnStatus = false;
     }
-    for (var i = 0; i < alength; i++) {
+    for (i = 0; i < alength; i++) {
```

Comment:
Use var, otherwise `i` goes into global scope.

Reply:
Wierd, wonder if that was my linter...  I'll make sure it's not doing that anymore.